deepchem/metrics/score_function.py

```diff
@@ -162,3 +162,54 @@ def bedroc_score(y_true: np.ndarray, y_pred: np.ndarray, alpha: float = 20.0):
   scores = sorted(scores, key=lambda pair: pair[1], reverse=True)
 
   return CalcBEDROC(scores, 0, alpha)
+
+
+def concordance_index(y_true: np.ndarray, y_pred: np.ndarray) -> float:
+  """Compute Concordance index.
+
+  Statistical metric indicates the quality of the predicted ranking.
+  Please confirm details from [1]_.
+
+  Parameters
+  ----------
+  y_true: np.ndarray
+    continous value
+  y_pred: np.ndarray
+    Predicted value
+
+  Returns
+  -------
+  float between [0,1]
```

Comment:
The docstring is not appropriate. The following is the correct style.
```
Returns
-------
int
    Description of anonymous integer return value.
```

Reply:
Ah, I checked.